apps/web/components/eventtype/EventAppsTab.tsx

```diff
@@ -64,9 +65,11 @@ export const EventAppsTab = ({ eventType }: { eventType: EventType }) => {
 
   const { shouldLockDisableProps, isManagedEventType, isChildrenManagedEventType } = useLockedFieldsManager(
     eventType,
-    t("locked_fields_admin_description"),
-    t("locked_fields_member_description")
+    methods,
+    t
   );
+  const appsDisableProps = shouldLockDisableProps("apps", { simple: true });
```

Comment:
`simple` flag makes the lock/unlock indicator to only show the switch which is the one needed for the alert